ci.nix

```diff
@@ -52,6 +52,9 @@
     # aarch64 cross only works on linux
     inherit (lib.systems.examples) musl64 aarch64-multiplatform;
   };
+  isDisabled = d:
+    let meta = d.meta or {};
+    in meta.disabled or false;
```

Comment:
This is a good idea - I was thinking about doing the same thing in cardano-wallet.